Dockerfile

```diff
@@ -94,9 +95,12 @@ COPY supervisor.conf /etc/supervisor/conf.d/
 
 # Entrypoint
 ADD start /app/bin/
-RUN chmod a+rx /app/bin/start
+COPY crontab.txt /app/crontab.txt
```

Comment:
As the file is no needed after installation, I'd rather copy it to /tmp what can be done together with `requirements.txt` few lines above.

Reply:
fixed